src/opnsense/mvc/app/library/OPNsense/Base/IDN.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
I don't think we should add a class for this, it doesn't really add functionality.